recap/converters/json_schema.py

```diff
@@ -65,12 +69,22 @@ def _parse(
             self.json_registry = resource @ self.json_registry
             extra_attrs["alias"] = alias_strategy(resource_id)
 
+        # Special handling for "type" defined as a list of strings like
+        # {"type": ["string", "boolean"]}
+        if "type" in json_schema and isinstance(json_schema["type"], list):
```

Comment:
I think this can be put inside the `match` statement. Something like:

```python
case {"type": list(type_list)}:
  types = [self._parse(s, alias_strategy) for s in type_list]
  return UnionType(types, **extra_attrs)
```

Reply:
Good call